frontend/src/components/applications/StaffingCertificateInput.vue

```diff
@@ -0,0 +1,198 @@
+<template>
+  <v-form ref="form">
+    <v-row no-gutters class="mt-8">
+      <v-col v-for="(certificate, index) in certificates" :key="index" cols="12" sm="6" lg="4" xl="3" class="px-6 px-lg-8 mb-2" align="center">
+        <v-row no-gutters>
+          <v-col cols="12" sm="1" class="pt-2">{{ index + 1 }}.</v-col>
+          <v-col cols="12" sm="4" class="pt-1">
+            <v-text-field
+              v-model.trim="certificate.initials"
+              variant="outlined"
+              density="compact"
+              :disabled="readonly"
+              placeholder="Initials"
+              maxlength="4"
+              rounded="0"
+              :rules="rules.required"
+              :error-messages="getErrorMessagesForDuplicateError(certificate.isInitialsDuplicate)"
+              @input="update(certificate)"
+              @blur="checkDuplicateInitials" />
+          </v-col>
+          <v-col cols="12" sm="6" class="pt-1">
+            <v-text-field
+              v-model="certificate.certificateNumber"
+              variant="outlined"
+              density="compact"
+              :disabled="readonly"
+              :placeholder="certificateNumberPlaceholder"
```

Comment:
The placeholder is being cut off:

![image](https://github.com/bcgov/ecc-ofm/assets/80127167/d12a0cb0-2733-41aa-ba1b-1bbf8c1f759f)

Reply:
yes, the placeholder is being cut off. If we want to always display the full placeholder text, we will need to reduce the number of columns, which will make the page really long when the provider has many staff (e.g.: 99 staffs --> 99 certificates). I will reduce the number of columns. If the clients complain about the length of the page, we can do an enhancement ticket to shorten the placeholder text or add a scrollbar to the certificates list.

Reply:
I've updated the placeholder text to "ECE certificate #"